components/modelregistry/modelregistry.go

```diff
@@ -0,0 +1,84 @@
+// Package modelregistry provides utility functions to config ModelRegistry, an ML Model metadata repository service
+package modelregistry
+
+import (
+	"context"
+	"path/filepath"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	dsciv1 "github.com/opendatahub-io/opendatahub-operator/v2/apis/dscinitialization/v1"
+	"github.com/opendatahub-io/opendatahub-operator/v2/components"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/deploy"
+)
+
+var (
+	ComponentName = "model-registry-operator"
+	Path          = deploy.DefaultManifestPath + "/" + ComponentName + "/overlays/odh"
```

Comment:
will you have different path for upstream ODH and downstream RHOAI in `overlays`?

Reply:
Is that needed? I thought that ODH/RHOAI differences were handled through different image names and image env variables.

Reply:
images for ODH and RHOAI are different, manifests can be exactly the same or totally different.
take dsp and  dashboard as 2 examples.
but if modelreg is decided to have this path, i am all fine with the code.